packages/button/README.md

```diff
@@ -128,10 +128,18 @@ mwc-button {
 ### CSS Custom Properties
 
 | Name | Default | Description
-| ---- | ------- | -----------
-| `--mdc-icon-font` | [`Material Icons`](https://google.github.io/material-design-icons/) | Font to use for the icon.
-| `--mdc-theme-primary` | ![](images/color_6200ee.png) `#6200ee` | Background color of the button
-| `--mdc-theme-on-primary` | ![](images/color_ffffff.png) `#ffffff` | Text color of the button
+| ------------------------------------- | -------------------------------------------------- | ---
+| `--mdc-icon-font` | [`Material Icons`](https://google.github.io/material-design-icons/)    | Font to use for the icon.
+| `--mdc-theme-primary`                 | ![](images/color_6200ee.png) `#6200ee`             | Background color of the button
+| `--mdc-theme-on-primary`              | ![](images/color_ffffff.png) `#ffffff`             | Text color of the button
+| `--mdc-button-text-transform`         | `uppercase`                                        | Sets the text-transform on the button label.
```

Comment:
I suggest removing all the "Sets the" prefixes, they aren't really necessary.

Reply:
Updated description to the same text for convenience.